.github/workflows/deploy.yml

```
@@ -0,0 +1,32 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pandoc/latex
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install mustache (to update the date)
+        run: apk add ruby && gem install mustache
+      - name: creates output
+        run: sh ./build.sh
+      - name: check dir
+        run : pwd
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)